Source/WebKit/UIProcess/Inspector/mac/WebInspectorUIProxyMac.mm

```diff
@@ -918,15 +918,17 @@ - (IBAction)_popUpButtonAction:(id)sender
 
 DebuggableInfoData WebInspectorUIProxy::infoForLocalDebuggable()
 {
-    NSDictionary *plist = adoptCF(_CFCopySystemVersionDictionary()).bridgingAutorelease();
+    CFDictionaryRef plist = _CFCopySystemVersionDictionary();
 
     DebuggableInfoData result;
     result.debuggableType = Inspector::DebuggableType::WebPage;
     result.targetPlatformName = "macOS"_s;
-    result.targetBuildVersion = plist[static_cast<NSString *>(_kCFSystemVersionBuildVersionKey)];
-    result.targetProductVersion = plist[static_cast<NSString *>(_kCFSystemVersionProductUserVisibleVersionKey)];
+    result.targetBuildVersion = static_cast<CFStringRef>(CFDictionaryGetValue(plist, _kCFSystemVersionBuildVersionKey));
+    result.targetProductVersion = static_cast<CFStringRef>(CFDictionaryGetValue(plist, _kCFSystemVersionProductUserVisibleVersionKey));
     result.targetIsSimulator = false;
 
+    CFRelease(plist);
```

Comment:
It is so trivial here though. Like, I am pretty sure a smart pointer here is WORSE because unlike other cases where the memory retention needs change over time, it is pretty clear what this is for and it will only be needed for this length of this function.

Not even worrying about multiple codepaths.